### PR TITLE
Zombie fast path w/ animation

### PR DESCRIPTION
**Pull request by isaacmorneau**
creating PR then im fixing the merge conflicts

### CODE REVIEW

**Comment by isaacmorneau** [comment]
just realized the number of magic numbers in here ill fix in the morning

File: src/basic/Movable.h [hunk 1]
```diff
@@ -26,25 +26,37 @@ class Movable : public virtual Entity {
...
-    void setAngle(const double a);//sets angle of sprite to
-    double getAngle() const; //returns sprites angle
+    // Moves Marine
+    void move(const float moveX, const float moveY, CollisionHandler& ch);
+    //set delta x coordinate
+    void setDX(float px) {dx = px;}
+    //set delta y coordinate
+    void setDY(float py) {dy = py;}
+    // set velocity of Marine movement
+    void setVelocity(int pvel) {velocity = pvel;}
```
Comment (JohnAgapeyev) [inline src/basic/Movable.h]: You're missing a const
Oh the travesty!
It's not 🌮 time
Please fix this spicy burrito

File: src/creeps/Zombie.cpp [hunk 2]
```diff
@@ -20,18 +20,18 @@
 #include <random>
 #include <cassert>
 #include <utility>
-
+#include <math.h>
```
Comment (JohnAgapeyev) [inline src/creeps/Zombie.cpp]: cmath

File: src/game/GameManager.h [hunk 3]
```diff
@@ -98,6 +98,7 @@ class GameManager {
 
     int32_t addObject(const Object&);
     void deleteObject(const int32_t id);
+    Object& getBase();
```
Comment (JohnAgapeyev) [inline src/game/GameManager.h]: Trista has a PR up that I think handles this
I'd suggest we get that merged in first, so that you're not redoing her work

**Review by JohnAgapeyev — CHANGES_REQUESTED** [review]
YOU'RE MISSING A CONST
THAT IS UNACCEPTABLE MISTER /s
Realistically, Trista has a PR up that redoes how bases are handled, and I think you are stepping on each other's toes.
So I'd get her PR merged in first before yours to ensure you're not redoing the same work

**Review by JohnAgapeyev — APPROVED** [review]

**Pull request merged**